scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/task/internal/InternalTask.java

```diff
@@ -1044,6 +1043,11 @@ protected TaskLauncherInitializer getDefaultTaskLauncherInitializer(InternalJob
         tli.setIterationIndex(getIterationIndex());
         tli.setReplicationIndex(getReplicationIndex());
 
+        HashMap<String, String> gInfo = new HashMap<>();
+        gInfo.putAll(job.getGenericInformation());
+        gInfo.putAll(getGenericInformation());
```

Comment:
So a task overwrites the generic information of a job. In order to make that obvious you could put that into a private method named with a description.